plugins/linux/ifplugin/descriptor/interface.go

```diff
@@ -260,6 +270,22 @@ func (d *InterfaceDescriptor) Validate(key string, linuxIf *interfaces.Interface
 		if linuxIf.GetVeth().GetPeerIfName() == "" {
 			return kvs.NewInvalidValueError(ErrVETHWithoutPeer, "peer_if_name")
 		}
+	case *interfaces.Interface_Loop:
+		nsCtx := nslinuxcalls.NewNamespaceMgmtCtx()
+		nsHandle, err := d.nsPlugin.GetNamespaceHandle(nsCtx, linuxIf.Namespace)
+		if err != nil {
+			return kvs.NewInvalidValueError(ErrNamespaceDoesntExists, "namespace")
+		}
+		nsHandle.Close()
+
+		// in each network namespace there is exactly one loopback interface,
+		// multiple configurations for a single namespaces is not valid
+		alias, err := d.getLoopbackAlias(nsCtx, linuxIf)
+		if err != nil || (alias != "" && alias != linuxIf.Name) {
+			d.log.WithField("error", err).Errorf("loopback already configured using logical name '%v'", alias)
```

Comment:
This error is I think also returned when the loopback doesn't exist (if it can even happen), would it be possible to differentiate these two cases?